.gitignore

```diff
@@ -11,7 +11,6 @@ dist/
 vendor/
 bin/
 .release-env
-*.tfstate*
```

Comment:
Removed this otherwise `plugins/source/terraform/resources/services/testdata/terraform.tfstate` is not committed.
I could make it more specific, but I don't think we need it since we removed the integration tests